src/components/FileInputMetadata.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import filesize from 'filesize';
+import { string, number } from 'prop-types';
+import '../styles/FileInputMetadata.scss';
+
+const styleName = name => {
+  // Do we have list of supported image types ??
```

Comment:
Yes, and that question should be included in comments in JIRA, not in code. For now, we can stay with `image/jpeg`, `image/gif` and `image/png`. Also, as you probably see now, we should check the mime type, not only a file extension.